luigi/parameter.py

```diff
@@ -893,6 +893,52 @@ def serialize(self, e):
         return e.name
 
 
+class EnumListParameter(Parameter):
+    """
+    A parameter whose value is a comma-separated list of :class:`~enum.Enum`.
+
+    In the task definition, use
```

Comment:
Also add one line explaining that this is a list in the sense that order matters and there can be duplicates and it can be empty.

Reply:
OK